src/buildExec.ts

```diff
@@ -7,7 +7,7 @@ import {setFailure} from './helpers';
 
 const context = github.context;
 
-const isTrue = (variable) => {
+const isTrue = (variable: string): boolean => {
```

Comment:
Add missing type declarations.
Note: Added return types to speed up the compile time (probably not noticeable, but still an improvements and makes it easier to spot mistakes)